docs/h5py_dataset.rst

```diff
@@ -57,15 +57,15 @@ things if your data happens to meet these assumptions:
   names.
 * Data sources are not explicitly split into separate HDF5 datasets or separate
   HDF5 files. Instead, splits are defined in the ``split`` attribute of the root
-  group. It's expected to be a 1D numpy array of compound ``dtype`` with six
+  group. It's expected to be a 1D numpy array of compound ``dtype`` with seven
   fields, organized as follows:
 
   1. ``split`` : string identifier for the split name
   2. ``source`` : string identifier for the source name
   3. ``start`` : start index (inclusive) of the split in the source array
   4. ``stop`` : stop index (exclusive) of the split in the source array
-  5. ``available`` : boolean, ``False`` if this split is not available for this
-     source
+  4. ``indices`` : reference pointing to a dataset containing indices for this split/source pair
+  5. ``available`` : boolean, ``False`` if this split is not available for this source
```

Comment:
The reader at this  point might be confusing by apparent overlapping between `start`, `stop` and `indices` roles. Perhaps an explanation would help.